lib/kitchen/driver/aws/client.rb

```diff
@@ -65,7 +64,7 @@ def self.get_credentials(profile_name, access_key_id, secret_access_key, session
               ENV["AWS_SECRET_ACCESS_KEY"],
               ENV["AWS_SESSION_TOKEN"]
             )
-          elsif shared_creds.loadable?
```

Comment:
This method is already called in the `Aws::SharedCredentials` constructor. Cf. http://docs.aws.amazon.com/sdkforruby/api/Aws/SharedCredentials.html (view source)